src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import static core.basesyntax.ColorSupplier.getRandomColor;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier.Colors randomColor = ColorSupplier.Colors.valueOf(getRandomColor());
```

Comment:
why do you convert String again to Colors, just use getRandomColor value directly